src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/sdk/Sdk.props

```diff
@@ -0,0 +1,17 @@
+<!--
+***********************************************************************************************
+Sdk.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project>
+
+  <!-- Only import the build props if the ILLink.Tasks package isn't referenced via NuGet. -->
+  <Import Project="$(MSBuildThisFileDirectory)..\build\Microsoft.DotNet.SDK.ILCompiler.targets" />
```

Comment:
Did you mean this file to be `Sdk.targets`? Importing a targets from a props file is kind of an antipattern